source/usage-examples/bulkWrite.txt

```diff
@@ -0,0 +1,86 @@
+=======================
+Perform Bulk Operations
+=======================
+
+.. default-domain:: mongodb
+
+Overview
+--------
+
+MongoDB provides clients the ability to perform write operations in bulk.
+Bulk operations affect a *single* collection.
+
+The following code sample performs a bulk write operation against the
+``users`` collection in the ``sample_mflix`` database. Specifically, the
+program uses the :node-api:`collection.bulkWrite
+<Collection.html#bulkWrite>` method and ``insertOne`` operations to write
+the user information from a ``users.json`` file to the ``users``
+collection. Download the dataset here: `users.json <https://raw.githubusercontent.com/mongodb-university/universal-driver-examples/master/users.json>`_
+
+``bulkWrite()`` accepts the following parameters:
+
+- ``operations`` (``Array.<object>``): specifies the bulk operations to
```

Comment:
While I like the fact that you document the type immediately after the parameter name, I think we're trying to keep the tone of these slightly less reference-y so I've omitted this level of detail in my own. I figure that since we're saying the same thing later on ("an object in an array") it's mostly redundant anyway.